.github/workflows/dart_ci.yml

```diff
@@ -11,6 +11,11 @@ on:
     branches:
       - '**'
 
+permissions:
```

Comment:
In the last release we hit this build error on the `anchore/sbom-action@v0` step:
```
------------------------- Uploading workflow artifacts -------------------------
/tmp/sbom-action-jo68vn/over_react-build.cyclonedx.json
Found 1 artifact(s)
---------------------- Attaching SBOMs to release: '5.3.2' ---------------------
/usr/bin/unzip -o -q /home/runner/work/_temp/e4596f5b-92f8-4f38-a5db-dbde4d5545d6
/home/runner/work/_temp/0f6b674c-ce8a-4b9b-842e-7ddb2a1e1cd3/over_react-build.cyclonedx.json
Error: Resource not accessible by integration
```
Which should hopefully be fixed by adding these permissions